src/VisualStudio/CSharp/Impl/LanguageService/CSharpHelpContextService.cs

```diff
@@ -206,6 +206,13 @@ private bool TryGetTextForSymbol(SyntaxToken token, SemanticModel semanticModel,
 
         private static bool TryGetTextForOperator(SyntaxToken token, Document document, out string text)
         {
+            if (token.IsKind(SyntaxKind.ExclamationToken) && token.Parent is not null &&
```

Comment:
The null check is not necessary because `IsKind` already checks for null.

```suggestion
            if (token.IsKind(SyntaxKind.ExclamationToken) &&
```